tests/Tests/Unit/GapicClientTraitTest.php

```diff
@@ -185,6 +183,7 @@ public function testStartOperationsCall()
         $message = new MockRequest();
         $operationsClient = $this->getMockBuilder(OperationsClient::class)
             ->disableOriginalConstructor()
+            ->setMethodsExcept(['validate'])
```

Comment:
This line is causing PHP 5.5 tests to fail (the `setMethodsExcept` method doesn't exist in, presumably, PHPUnit 4.x). 

I imagine we could work around this somehow? By mocking `validate` to return null, or something along those lines?

I hate to spin wheels supporting PHP 5.5 though, so another option is to wait and we can drop it soon (within a week).

Reply:
Dropping support for PHP 5.5 would probably be the easiest, but we could also du a follow-up release. It would be nice not having to wait though (also the challenge 😄)

I will try to find PHP 5.5 somewhere so I can test it before pushing the branch just to find out it's still failing. 🤞

Reply:
It would be great to get a work-around for this so it isn't blocked by dropping PHP 5.5 support! That is, assuming it's an easy fix 🤞

Reply:
Okay, I didn't get a hand on PHP 5.5, but I applied your suggestion, and it seems to work (locally). Let's see what GitHub Actions says when I push this. 🤞